README.md

```diff
@@ -383,5 +384,16 @@ epoch runs log [-h] topology-id run-id task-name
 `run-id` - Run-id 
 `task-name` - Task-name to be fetched
 
+## Developer-setup 
+```
+python3 -m venv testingEpochCli
+cd testingEpochCli
+source bin/activate
+pip install epoch-cli
+```
+Move the setting.ini to the testingEpochCli directory and add your valid olympus tokens along with valid stage and prod urls.
```

Comment:
this is very phonepe specific. rephrase to: auth tokens/username/passwords along with valid environments for different epoch endpoints.